repository/src/main/java/org/alfresco/repo/event2/NodeResourceHelper.java

```diff
@@ -158,6 +160,15 @@ private boolean isSubClass(QName className, QName ofClassQName)
         return dictionaryService.isSubClass(className, ofClassQName);
     }
 
+    private String getAssocQName(NodeRef nodeRef) {
```

Comment:
Please use Alfresco format. 
Also, it makes sense to rename the method or add JavaDoc to indicate this only returns the primary parent-child association.

Reply:
@killerboot wrote a similar comment, Can I change the name to primaryAssocQName in order to clarify it?